lazer/contracts/evm/test/PythLazer.t.sol

```diff
@@ -20,4 +20,40 @@ contract PythLazerTest is Test {
         skip(2000);
         assert(!pythLazer.isValidSigner(address(2)));
     }
+
+    function test_verify_with_fee() public {
+        // Prepare dummy update and signer
+        address trustedSigner = 0xEfEf56cD66896f6799A90A4e4d512C330c094e44;
+        vm.prank(address(1));
+        pythLazer.updateTrustedSigner(trustedSigner, 3000000000000000);
+        bytes memory update = hex"2a22999a577d3cc0202197939d736bc0dcf71b9dde7b9470e4d16fa8e2120c0787a1c0d744d0c39cc372af4d1ecf2d09e84160ca905f3f597d20e2eec144a446a0459ad600001c93c7d3750006240af373971c01010000000201000000000005f5e100";
+
+        uint256 fee = pythLazer.verification_fee();
+
+        address alice = makeAddr("alice");
+        vm.deal(alice, 1 ether);
+        address bob = makeAddr("bob");
+        vm.deal(bob, 1 ether);
+
+        // Alice provides appropriate fee
```

Comment:
I decided to just bunch these all together. I can also make them separate unit tests too, but I liked the idea of making more than one call on an account and confirming their balance changes as expected.